src/proto/index.js.flow

```diff
@@ -81,16 +81,21 @@ declare export class Address {
     }): Address;
     toJSON(): Object;
 }
+type AccountDetails = {
+    identifier: string;
+    type: 'INVALID' | 'OTHER' | 'CHECKING' | 'SAVINGS' | 'LOAN';
+    status: string;
+    metadata: Object;
+}
 declare export class Account {
     id: string;
     name: string;
     bankId: string;
-    tags: Array<any>;
     isLocked: boolean;
     accountFeatures: any;
     lastCacheUpdateMs: string;
     nextCacheUpdateMs: string;
-    accountDetails: any;
+    accountDetails: ?AccountDetails;
```

Comment:
do we need this `?`?

Reply:
Isnt it optional? It says its optional in the proto file, as a comment

Reply:
I am apparently no expert, I thought this line https://github.com/tokenio/sdk-js/pull/151/files#diff-930e0482a762aae9807a3c9ceeaab71eL103 indicated that it was optional?

Reply:
I meant line 108 below

Reply:
they both signify optionality, the 108 one means you dont have to pass it when creating, and this one means if you do account.accountDetails it might return undefined.